include/bluetooth/services/ias.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_IAS_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_IAS_H_
+
+/**
+ * @brief Immediate Alert Service (IAS)
+ * @defgroup bt_ias Immediate Alert Service (IAS)
+ * @ingroup bluetooth
+ * @{
+ *
+ * [Experimental] Users should note that the APIs can change
+ * as a part of ongoing development.
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief Immediate Alert Service callback structure. */
+struct bt_ias_cb {
+	/**
+	 * @brief Callback function to stop alert.
+	 *
+	 * This callback is called when peer commands to disable alert.
+	 */
+	void (*stop_alert)(void);
+
+	/**
+	 * @brief Callback function for alert level value.
+	 *
+	 * This callback is called when peer commands to alert.
+	 */
+	void (*start_alert)(void);
+
+	/**
+	 * @brief Callback function for alert level value.
+	 *
+	 * This callback is called when peer commands to alert in the strongest possible way.
+	 */
+	void (*start_alert_high)(void);
+};
+
+/**
+ * @brief Registers callbacks for ias_client.
+ *
+ * @param cb   Pointer to the callback structure.
+ *
+ */
+int bt_ias_cb_register(struct bt_ias_cb *cb);
```

Comment:
Implementation missing

Reply:
I don't think this is even necessary, as BT_IAS_CB_DEFINE is used for registering callbacks.